mne/viz/ica.py

```diff
@@ -947,6 +947,7 @@ def _close_event(events, params):
 def _plot_sources_epochs(ica, epochs, picks, exclude, start, stop, show,
                          title, block):
     """Plot the components as epochs."""
+
```

Comment:
This is a `docstyle` error :)

But other than this everything passes style tests